README.md

```diff
@@ -24,25 +23,37 @@ To use the Draft Web Design Standards on your project, you’ll need to include
 
 First, download the Draft Web Design Standards assets:
 
-[https://github.com/18F/web-design-standards/archive/v0.8.3.zip](https://github.com/18F/web-design-standards/archive/v0.8.3.zip).
+[https://github.com/18F/web-design-standards/releases/download/v0.9.0/uswds-0.9.0.zip](https://github.com/18F/web-design-standards/releases/download/v0.9.0/uswds-0.9.0.zip)
 
 Then, add the `dist` directory files into a relevant place in your code base — likely a directory where you keep third-party libraries:
 
 ```
-- uswds
-  - fonts/
-  - img/
-  - js/
-  - css/
+- uswds-0.9.0/
+  - dist/
+    - css/
+    - fonts/
+    - img/
+    - js/
 ```
 
-Refer to these files by adding a `<link>` and a `<script>` element into your HTML pages:
+Refer to these files by adding the following `<link>` and `<script>` elements into your HTML pages.
```

Comment:
I'd add a colon after "page" (because of the preceding "adding the following link").